util/container.go

```diff
@@ -52,6 +52,17 @@ func ContainerRuntimeInit(ctx context.Context, clusterName string) *client.Clien
 	spinr.FinalMSG = "done!"
 
 	for _, baseVitessContainer := range globals.VitessContainers {
+    if baseVitessContainer.Name =="mysqld" && globals.MysqlFlavor != "mysql" {
```

Comment:
Spacing here is off :)

Reply:
That is weird, my editor doesn't look like that....

Reply:
However when I took that section and copy/pasted it somewhere else the weird spacing issue appeared, I erased that line, hit enter, let my editor reformat it; then copy/paste checked it. 

```
	spinr.FinalMSG = "done!"

	for _, baseVitessContainer := range globals.VitessContainers {
		if baseVitessContainer.Name =="mysqld" && globals.MysqlFlavor != "mysql" {
			if globals.MysqlFlavor == "percona" {
```